test/integration/locale_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class LocaleTest < ActionDispatch::IntegrationTest
```

Comment:
I would be very careful naming test classes. If both classes have same name, they will be "merged", which will in turn lead to sporadic, extremely hard-to-find errors (say if you defined some setups/teardowns).